### ISSUE TITLE
Build fails on encountering non-string node value

### ISSUE THREAD

**Issue by Antontelesh** [postcss/postcss-simple-vars#14]
I'm using this plugin with webpack.
It fails to build when encounters numeric node value.
I haven't managed to track which exact selector or rule caused this, but it throws
```
Module build failed: TypeError: undefined is not a function
    at /Users/antontelesh/test_coding/book-library-angular/node_modules/postcss-simple-vars/index.js:71:40
    at /Users/antontelesh/test_coding/book-library-angular/node_modules/postcss/lib/container.js:104:26
    at Rule.each (/Users/antontelesh/test_coding/book-library-angular/node_modules/postcss/lib/container.js:91:22)
    at Rule.eachInside (/Users/antontelesh/test_coding/book-library-angular/node_modules/postcss/lib/container.js:103:21)
```

Issue can be resolved by placing `toString()` at `index.js:71:40`:
```
} else if ( node.value.toString().indexOf('$') != -1 ) {
```

P.S. Debugging has shown that build fails on `margin-right: 0;` where `typeof node.value === number`.

I can create PR a bit later if you wish.

**Comment by ai** [comment]
Seems like some other plugin save number to `decl.value`.

1. Yes, can you send a PR with `toString()`. Let’s make this plugin more bullet-proof.
2. What is our plugins list? We should fix that evil plugin too.

**Comment by Antontelesh** [comment]
I've sent you a PR.

And here is my full postcss config section of webpack:
```
postcss: [
      require('autoprefixer-core'),
      require('postcss-mixins')({ mixins: require('postcss-neat')() }),
      require('postcss-nested'),
      require('postcss-simple-vars')
    ],
```

**Comment by ai** [comment]
Source of problem was in `postcss-neat`, but i fixed it in `postcss-mixins` https://github.com/postcss/postcss-mixins/commit/f40b1431bfcd34ccbf100006a5f4c6876a6b676c

**Comment by ai** [comment]
Release will be only on this Mayy weekend :(. But you can use git version before release.

**Comment by Antontelesh** [comment]
Thanks a lot!

**Comment by ai** [comment]
Released in 0.3 version.

**Comment by Antontelesh** [comment]
Great! Thank you for notification :)
